kernel_hardening_checker/__init__.py

```diff
@@ -36,7 +36,7 @@ def _open(file: str) -> TextIO:
         sys.exit(f'[!] ERROR: unable to open {file}, are you sure it exists?')
 
 
-def detect_arch(fname: str, supported_archs: List[str]) -> Tuple[StrOrNone, str]:
+def detect_arch_kconfig(fname: str, supported_archs: List[str]) -> Tuple[StrOrNone, str]:
```

Comment:
Please add "in kconfig" to the error messages.

It's needed for understanding which of `detect_arch_*()` functions failed.